pkg/scheduler/scheduler.go

```diff
@@ -427,7 +427,7 @@ func (s *Scheduler) admit(ctx context.Context, e *entry) error {
 		PodSetAssignments: e.assignment.ToAPI(),
 	}
 
-	workload.SetAdmission(newWorkload, admission)
+	workload.SetQuotaReservation(newWorkload, admission)
```

Comment:
Let's set the Admitted condition here if the ClusterQueue doesn't have any checks. I guess it might be enough to call SyncAdmittedCondition.

Reply:
it's done in `SetQuotaReservation`